packages/components/src/utils/hooks.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Should there be tests and/or documentation for this? It took me some time to understand its purpose, even with the JSDoc for the component function.

Reply:
Sure thing!! I'll add some tests 😊 
Really appreciate your impressions on this!

Reply:
Thanks! Tests are of course good to have in any case. In this instance, it was largely because I can sometimes rely on the test case descriptions to inform me of what's expected from the behavior of the utility. It's much clearer now.